src/web/components/ClickToView.tsx

```diff
@@ -209,6 +196,7 @@ export const ClickToView = ({
 						icon={<SvgCheckmark />}
 						iconSide="left"
 						onClick={() => handleClick()}
+						data-cy="click-to-view-button"
```

Comment:
As above, we can remove these

Reply:
I think we need this one, but I can remove the ```data-cy="click-to-view"``` on the component.

Reply:
Ah, yes, you're right, they're different 👍